### DISCUSSION TITLE
How can I do a mass-resend of all "soft-fail" messages?

### DISCUSSION THREAD

**Discussion by chrism417** [postalserver/postal#2080] [Help with using Postal]
We had a mail server go down in development and no one realized it. Now I have 11k emails with "SOFT FAIL"

Is there someway to resend all instead of checking one by one?

**Comment by willpower232** [comment]
Do you mean Postal was not able to send the message on to your own mail server? You might be able to poke them via the database and console but I'm not sure off the top of my head.

**Reply by chrism417** [reply]
Hello, correct our internal mail server was down. I know we can do it 1x1 in the console but not sure how to do them all at once or in bulk

**Comment by chrism417** [comment]
@willpower232 any luck on this? I haven't found a way yet

**Comment by chrism417** [comment]
I found a way to retry messages in the queued_messages table, but nothing yet in the 'postal-server-1.messages' table.

For the queued_messages table:
`UPDATE postal.queued_messages SET locked_by = null, locked_at = null, retry_after = DATE_ADD(NOW(), INTERVAL 20 SECOND) WHERE domain = '<domainhere>';`
